documentation/guides/draft/rust/get-started/xx-cloud-node/README.md

```diff
@@ -0,0 +1,95 @@
+# Using Cloud nodes
+
+## Ockam Hub
+
+In previous examples we learned how to send messages between nodes using [transports](../07-routing-over-transport).
+
+But to connect two nodes at least one of them must be exposed via hostname or a public IP, which makes connecting devices challenging. To address that we can use special kind of node running on the cloud with a fixed hostname and make it forward messages between devices. We call such nodes Ockam Hub Nodes.
+
+You can host a Hub Node yourself, or use one provided by Ockam service at https://hub.ockam.network
+
+Hub Nodes run persistent workers, which are called services. Services can be used for discovery, routing and
+integration with various cloud services used by the application.
+
+This guide shows how to start a new Hub Node, connect an application and use services in there.
+
+## Creating Hub Nodes
+
+1. Navigate to http://hub.ockam.network
+
+1. In order to create a node, you need to log in using your GitHub account:
+
+    <img src="./Github-log-in.png" width="100%">
+
+1. After that you can create a routing node:
+
+    <img src="./Create-node.png" width="100%">
+
+1. When the node status changes to `Running`, the node is ready to use.
+
+1. You can copy the node name from the nodes list and use in the following example.
+
+    <img src="./Node-hostname.png" width="100%">
+
+## Example service usage
+
+In this example we're going to use the `echo_service` on the Hub Node we created. This service behaviour is similar to the `echoer` workers we used before - it will reply for a message with the same payload.
+
```

Comment:
```suggestion
In this example, we're going to move our previous `echo_service` to a Hub Node, The service behavior is the same - it will reply to a message with the same payload.
```